UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentAncillaryFilesTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using FakeItEasy;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.Common.Configuration;
+using UKHO.ExchangeSetService.Common.Helpers;
+using UKHO.ExchangeSetService.Common.Models.FileShareService.Response;
+using UKHO.ExchangeSetService.FulfilmentService.Services;
+
+namespace UKHO.ExchangeSetService.Webjob.UnitTests.Services
+{
+    [TestFixture]
+    public class FulfilmentAncillaryFilesTest
+    {
+        public IOptions<FileShareServiceConfiguration> fakeFileShareServiceConfig;
+        public IFileSystemHelper fakeFileSystemHelper;
+        public FulfilmentAncillaryFiles fulfilmentAncillaryFiles;
+        public string batchId = "63d38bde-5191-4a59-82d5-aa22ca1cc6dc";
```

Comment:
change variable names from batchId to fakeBatchId and exchangeSetRootPath to fakeExchangeSetRootPath

Reply:
Changes done for variable names.